content/learn/overview.md

```diff
@@ -20,36 +26,129 @@ layout: "overview"
+
+<p>The strength and success of GRASS GIS relies on the user community. 
+The *philosophy* of the GRASS GIS Development Team is to encourage users 
+to develop their own unique tools and applications as well as to improve 
+the existent ones.</p>
 
-<p>The strength and success of GRASS GIS relies on the user community. The philosophy of the GRASS GIS Development Team is to encourage users to develop their own unique tools and applications as well as to improve the existent ones.</p>
 </div>
 <div class="col-lg-6 text-center">
 <img class="bsh" src="/images/gallery/gui/grass70_gui_example.png" width="89%" alt="GRASS GIS">
 </div>
+
 </div>
 </div>
 
 ### Main features
 
 <div class="container">
 <div class="row mt-20">
+
 <div class="col-lg-6 text-center">
 <img class="bsh" src="/images/gallery/gui/gui-grass-example-2.png" width="89%" alt="GRASS GIS">
 </div>
+
 <div class="col-lg-6">
-<p><b>GRASS GIS</b> runs on Linux, Mac and Windows. It comes with both an intuitive graphical user interface and a command line syntax usefull for automation and production.</p>
-<p><b>GRASS GIS</b> contains over <b><a href="https://grass.osgeo.org/grass-devel/manuals/full_index.html" target="_blank">500 modules</a></b> to process and render geographic data. The software allows to manipulate a variety of raster, vector and 3D formats, and run simple to advanced spatial analysis and modeling. GRASS GIS can connect to spatial databases but also interface with a variety of third-party systems and libraries.</p>
-<p><b>GRASS GIS</b> capabilities are significantly increased by an impressive list of <b>extensions or add-ons</b>. These modules are contributed by users or developers but are not yet part of the standard distribution.
-  There are currently more than <b>300 extensions</b> in the <a href="https://github.com/OSGeo/grass-addons/tree/master/grass7" >official GRASS Add-ons repo</a> and many others in the wild to perform the most varied type of tasks.
-                    <br>Have a look at the full list of <a href="https://grass.osgeo.org/grass7/manuals/addons/" target="_blank">add-ons manual pages</a> to get an idea. If you don't find what you need, maybe you can <a href="/contribute/development/">develop</a> your own add-on then.
-                    <br>Read <a href="/download/addons/">here</a> to learn how to install them.</p>
+<p><b>GRASS GIS</b> runs on Linux, Mac and Windows. It comes with 
+both an intuitive graphical user interface (GUI) and a command line 
+syntax useful for automation and production.</p>
+<p><b>GRASS GIS</b> contains over 
+<b><a href="https://grass.osgeo.org/grass-devel/manuals/full_index.html" target="_blank">500 modules</a></b> 
+to process and render geographic data. The software allows to manipulate
+a variety of raster, vector and 3D formats, and run simple to advanced 
+spatial analysis and modeling. GRASS GIS can connect to spatial databases 
+but also interface with a variety of third-party systems and libraries.</p>
```

Comment:
Mention Jupyter notebooks here?

Reply:
It's below in the Interfaces section. I can add a `See below` sentence.